docker-compose-deploy.yaml

```diff
@@ -1,7 +1,7 @@
 version: '2.2'
 
 x-jellyfish-template: &jellyfish-template
-  image: "ghcr.io/jellyfish-dev/jellyfish:${TAG:-0.2.0}"
+  image: "ghcr.io/jellyfish-dev/jellyfish:${JELLYFISH_VERSION}"
```

Comment:
I also removed the default value here as in deploy compose env JELLYFISH_VERSION should always be present